packages/run-protocol/test/test-gov-collateral.js

```diff
@@ -0,0 +1,452 @@
+// @ts-check
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import process from 'process';
+import url from 'url';
+import path from 'path';
+import { E, Far } from '@endo/far';
+import {
+  addBankAssets,
+  makeAddressNameHubs,
+  makeOracleBrands,
+  makeBoard,
+  startPriceAuthority,
+} from '@agoric/vats/src/core/basic-behaviors.js';
+import {
+  bridgeCoreEval,
+  makeClientManager,
+} from '@agoric/vats/src/core/chain-behaviors.js';
+import { extractCoreProposalBundles } from '@agoric/deploy-script-support/src/extract-proposal.js';
+import { makeCoreProposalBehavior } from '@agoric/deploy-script-support/src/coreProposalBehavior.js';
+import { makeNameHubKit } from '@agoric/vats/src/nameHub.js';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+import { makeNodeBundleCache } from './bundleTool.js';
+import { setupBootstrap, setUpZoeForTest } from './supports.js';
+
+const dirname = url.fileURLToPath(new URL('.', import.meta.url));
+
+/** @type {import('ava').TestInterface<Awaited<ReturnType<makeTestContext>>>} */
+// @ts-expect-error cast
+const test = anyTest;
+
+const contractRoots = {
+  mintHolder: '../vats/src/mintHolder.js',
+  econCommitteeCharter: './src/econCommitteeCharter.js',
+};
+
+const coreProposals = {
+  addCollateral: '../scripts/add-collateral-core.js',
+  startRunPreview: '../scripts/init-core.js',
+  inviteCommittee: '../scripts/invite-committee-core.js',
+};
+
+const voterAddresses = {
+  Rowland: `agoric1qed57ae8k5cqr30u5mmd46jdxfr0juyggxv6ad`,
+  Bill: `agoric1xgw4cknedau6xhrlyn6c8e40d02mejee8gwnef`,
+  Dan: `agoric1yumvyl7f5nkalss7w59gs6n3jtqv5gmarudx55`,
+};
+
+const makeTestContext = async () => {
```

Comment:
this is only used within `test.before`. consider inlining it to avoid footgun of calling it within a test.

Reply:
its type is used everywhere...

```js
/** @type {import('ava').TestInterface<Awaited<ReturnType<makeTestContext>>>} */
// @ts-expect-error cast
const test = anyTest;
```